ui/index.html

```diff
@@ -1517,7 +1882,7 @@ <h5 class="modal-title">Connect wallet</h5>
                     explorer: 'https://explorer.testnet.rsk.co',
                     explorerTokenTab: '?__tab=tokens%20transfers',
                     confirmations: 10,
-                    confirmationTime: '5 minutes',
+                    confirmationTime: '50 seconds',
```

Comment:
in RSK as every block is 30 seconds it's ok to be 5 minutes, i think you where trying to modify the kovan parameter